clang/utils/reduce-clang-crash.py

```diff
@@ -186,22 +237,30 @@ def skip_function(func_name):
 
         self.expected_output = result
 
-    def check_expected_output(self, args=None, filename=None):
+    def check_expected_output(self, cmd=None, args=None, filename=None):
+        if not cmd:
```

Comment:
having all these default arguments makes it very hard to reason, it would be nice to clean up the callers to explicitly pass things in

Reply:
ack. I'll look at doing that.